research/audioset/vggish/vggish_slim.py

```diff
@@ -31,28 +31,31 @@
 """
 
 import tensorflow.compat.v1 as tf
-tf.disable_v2_behavior()
 import tf_slim as slim
 
 import vggish_params as params
 
 
-def define_vggish_slim(training=False):
+def define_vggish_slim(features=None, training=False):
```

Comment:
When I read this, I assumed it was a way to pass in actual values.  You could call it features_tensor.  In the context of the original function, it was obviously a tensor, but promoted to visibility outside the function, it's more ambiguous.

Reply:
Done